build.sbt

```diff
@@ -5,7 +5,7 @@ lazy val core = project.in(file("."))
     )
 
 val catsV = "1.4.0"
-val catsEffectV = "1.0.0"
+val catsEffectV = "1.1.0-M1"
```

Comment:
Hmm, this is a bit problematic. I gues this could be another milestone and we wait till this is fully released?

Reply:
Why? It is binary compatible with `1.0.0` right?

Reply:
Having the summoners of `Clock` and `Timer` is such a nice thing to have...

Reply:
I try not to release stable versions on milestones. I agree that those are nice and would be nice, perhaps we can wait for 1.1.0 to do a solid release.

Reply:
Okay, I get your point :+1: